wellcomecollection/wellcomecollection.org
# Featured card refactor

## What does this change?

Prep work for adding FeaturedCards on stories to represent articles and exhibitions.
I wanted to add those alternatives to Cardigan and have done so.
I've also refactored the component so instead of exporting three different components (`FeaturedCard`, `FeaturedCardExhibition` and `FeaturedCardArticle`), it would export the one (`FeaturedCard`) and if you wanted to specify if it was an article or an exhibition you could. I've also moved the styles and the helpers out as it was quite a complex file at the start. Happy to move them back in if we'd rather.

Would it be worth adding a snapshot for each variable (I haven't looked into how)?

## How to test

Play with it here: https://62f13cdbd0ff140768a8d87b-lsxwrfyaot.chromatic.com/?path=%2Fstory%2Fcomponents-cards-featuredcard--basic&globals=viewport.width%3A1200
Run Cardigan locally.
Run the website locally and go see the one current instance of the alternatives cards (on `/stories` and `/whats-on`).

## How can we measure success?

Clearer use of component allowing faster work moving forward

## Have we considered potential risks?
N/A
